.golangci.yml

```diff
@@ -20,21 +20,7 @@ linters:
 linters-settings:
   goimports:
     local-prefixes: github.com/NVIDIA/k8s-device-plugin
-
-issues:
-  exclude-rules:
-  # We use math/rand instead of crypto/rand for unique names in e2e tests.
-  - path: tests/e2e/
-    linters:
-    - gosec
-    text: "G404"
-  # We disable the memory aliasing checks in tests
-  - path: ".*_test.go"
-    linters:
-    - gosec
-    text: "G601: Implicit memory aliasing in for loop"
-  # We create world-readable files in tests.
-  - path: ".*_test.go"
-    linters:
-    - gosec
-    text: "G306: Expect WriteFile permissions to be 0600 or less"
\ No newline at end of file
+  gosec:
+    excludes:
+      - G306 # We create world-readable files in tests.
```

Comment:
Does this not mean that we're now ignoring these globally? Why not limit them to tests explicitly?

Reply:
ah that's a fair point

Reply:
I guess I'll have to revert this change

Reply:
Plase check now